content/docs/04-clusters/01-public-cloud/01-aws/08-create-cluster.md

```diff
@@ -0,0 +1,193 @@
+---
+title: "Create and Manage AWS IaaS Cluster"
+metaTitle: "Create and Manage AWS Cluster"
+metaDescription: "Learn how to add and manage a cluster deployed to AWS."
+hideToC: false
+fullWidth: false
+category: ["how-to"]
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+import PointsOfInterest from 'shared/components/common/PointOfInterest';
+
+# Create and Manage AWS IaaS Cluster
+
+Palette supports creating and managing Kubernetes clusters deployed to an AWS account. This section guides you on how to create a Kubernetes cluster in AWS that is managed by Palette.
```

Comment:
`Palette supports creating and managing Kubernetes clusters deployed to an AWS account. This section guides you on how to create a Kubernetes cluster in AWS that Palette manages.`